src/main/kotlin/io/github/mojira/arisa/modules/PrivacyModule.kt

```diff
@@ -12,7 +12,8 @@ class PrivacyModule(
 ) : Module {
     private val patterns: List<Regex> = listOf(
         """\(Session ID is token:""".toRegex(),
-        """(?<!\[~)\b[a-zA-Z0-9.-_]+@[a-zA-Z0-9.-_]+\.[a-zA-Z0-9.-_]{2,}\b(?!])""".toRegex()
+        """(?<!\[~)\b[a-zA-Z0-9.-_]+@[a-zA-Z0-9.-_]+\.[a-zA-Z0-9.-_]{2,}\b(?!])""".toRegex(),
+        """--accessToken ey""".toRegex()
```

Comment:
Does the access token always start with `ey`? Isn't it possible that it starts with something else?

Reply:
Its a JWT token. They are 3 base64-encoded jsons. ey translates to { which must be the first character of the header(first part)

Reply:
Ah yeah, makes sense 👍